sdks/python/apache_beam/runners/interactive/README.md

```diff
@@ -237,11 +237,13 @@ You can choose to run Interactive Beam on Flink with the following settings.
     to the pipeline as follows.
 
     ```python
-    options = pipeline_options.PipelineOptions()
-    options.view_as(pipeline_options.PortableOptions).job_endpoint = 'localhost:8099'
-    options.view_as(pipeline_options.SetupOptions).sdk_location = 'container'
-    options.view_as(pipeline_options.DebugOptions).experiments = 'beam_fn_api'
-
+    options = pipeline_options.PipelineOptions([
```

Comment:
Better to write PipelineOptions(runner='PortableRunner', ...)

Reply:
Thanks! Applied.